general/project/projectinit.go

```diff
@@ -134,6 +137,24 @@ func (pic *ProjectInitCommand) createBuildMessage(technologiesMap map[coreutils.
 	return message
 }
 
+// Returns all detected technologies found in the project directory.
+// First, try to return only the technologies that detected according to files in the root directory.
+// In case no indication found, the search continue recursively.
```

Comment:
// In case no indication found, the search continue recursively.
-->
// In case no indication found in the root durectory, the search continue recursively.